packages/react/src/components/Authenticator/ResetPassword/ConfirmResetPassword.tsx

```diff
@@ -76,7 +76,7 @@ export const ConfirmResetPassword = (): JSX.Element => {
         <Header />
 
         <Flex direction="column">
-          <ConfirmationCodeInput />
+          <ConfirmationCodeInput type="number" />
```

Comment:
As discussed, changing code input type to "number" here. Will create a separate PR for enforcing it across all instances.